bin/ridgepole

```diff
@@ -86,6 +86,18 @@ ARGV.options do |opt|
     opt.on('-f', '--file SCHEMAFILE') { |v| file = v }
     opt.on('',   '--dry-run') { options[:dry_run] = true }
     opt.on('',   '--table-options OPTIONS') { |v| options[:table_options] = v }
+    opt.on('',   '--table-hash-options JSON') do |v|
+      json = JSON.parse(v, symbolize_names: true)
```

Comment:
How about using YAML?

* Compatible with JSON
* Same as the existing option (e.g. `--config '{adapter: mysql2...}'`)
* Can be written simply

Reply:
fixed 👍